content/app_changes.tex

```diff
@@ -126,6 +127,20 @@ \subsection*{Official Rules - Disqualification during competition}
   \item Added possibility to disqualify a team during competition for serious ethical breaches, or violation of the terms of their qualification~(\cf \cref{sec:disqualification_during_comp})
 \end{itemize}
 
+\subsection*{Official Rules - Championship Cup and Challenge Shield}
```

Comment:
We agreed on adding just the 5 to 10 major changes to the change list. Because the rest can be found in the repo

Reply:
OK...